Nos/Views/ProfileTab.swift

```diff
@@ -28,6 +31,18 @@ struct ProfileTab: View {
                         ProfileView(author: profile)
                     }
                 }
+                .task {
```

Comment:
Could we refactor this into a view modifier? It seems like the main part that isn't repeated is the name of path. So perhaps something like this would work:
```
extension View {
    func doubleTapToPop(path: Binding<NavigationPath>, isVisible: Binding<Bool>) -> some View {
...
}

Reply:
Refactored it in a ViewModifier instead bc I needed a State variable for storing the cancellable. It looks much better now, much less code than before.